compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -608,6 +615,32 @@ internal class RootNodeOwner(
             platformContext.setPointerIcon(desiredPointerIcon ?: PointerIcon.Default)
         }
     }
+
+    /**
+     * Implements [DragAndDropManager] by delegating to [PlatformDragAndDropManager].
+     */
+    private class PlatformDelegatingDragAndDropManager(
```

Comment:
It looks like it's more related to `PlatformDragAndDropManager` and needs to be placed there.

My vote is for helper `toDragAndDropManager()` function in `PlatformDragAndDropManager.kt` file

Reply:
I could do `PlatformDragAndDropManager.asDragAndDropManager()`, although I don't feel either way is better.

Reply:
My reasoning here - `RootNodeOwner` is already large enough, so I'd avoid adding here delegates that might be placed outside

Reply:
Done